<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRechercherUnClient</class>
 <widget class="QDialog" name="DialogRechercherUnClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rechercher un client</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sexe</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="Masculin">
        <property name="text">
         <string>Masculin</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="Feminin">
        <property name="text">
         <string>Féminin</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNom"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableView" name="tableViewClients">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>37</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Action :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxAction">
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContents</enum>
      </property>
      <item>
       <property name="text">
        <string>occuper une chambre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>réserver une chambre</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="btn_OK">
      <property name="maximumSize">
       <size>
        <width>95</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Exécuter action</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Annuler">
      <property name="maximumSize">
       <size>
        <width>95</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Fermer</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
